rust/ql/test/library-tests/dataflow/models/models.ql

```diff
@@ -15,66 +15,6 @@ query predicate invalidSpecComponent(SummarizedCallable sc, string s, string c)
   Private::External::invalidSpecComponent(s, c)
 }
 
-private class SummarizedCallableIdentity extends SummarizedCallable::Range {
```

Comment:
Are we retaining the ability to write models in QL as well as MAD?  (if so, we should retain at least one test case written this way)

In Swift I preferred to write the bulk of models in MAD.  But QL was useful for heuristic models that matched wide patterns - and may be necessary in places to fit in with the "concepts" framework???

Reply:
Yes, I can revert one of the tests to use QL.